src/commcare_cloud/ansible/roles/cloudwatch_logs/tasks/main.yml

```diff
@@ -22,16 +22,12 @@
       owner: cwagent
       mode: 0644
     become: yes
-    register: cloudwatch_logs_conf
-
-  - name: Reload cloudwatch agent if changed
-    shell: /opt/aws/amazon-cloudwatch-agent/bin/amazon-cloudwatch-agent-ctl -a fetch-config -m ec2 -s
-      -c file:/opt/aws/amazon-cloudwatch-agent/etc/cloudwatch_logs.json
-    become: yes
-    when: cloudwatch_logs_conf.changed
+    notify: Reload cloudwatch agent
 
   - name: Ensure cloudwatch agent enabled
-    shell: /opt/aws/amazon-cloudwatch-agent/bin/amazon-cloudwatch-agent-ctl -a start
-    become: yes
+    shell: /bin/true
```

Comment:
Doesn't this mean it always restarts? did you mean to do a status check?

Reply:
The check is on this line:

`when: '"\"status\": \"running\"" not in ctl_status.stdout'`

Calling /bin/true is just a way to always notify the handler when the test passes.

Reply:
Ahh I see you're checking the `ctl_status` from before. That makes sense and LGTM.